File: Assignments/MS.Net/05_08.12.2020/Assignment2/EmployeeAddressWebApp/Models/Employee.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace EmployeeAddressWebApp.Models
{
    public class Employee
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Gender { get; set; }
        public string Department { get; set; }
        public decimal Salary { get; set; }
        public int AddressId { get; set; }
    }
}

File: Assignments/JAVA/07_18-08-2020/src/com/app/org/Employee.java
package com.app.org;

public class Employee {
	private int id;
	private String name;
	
	public Employee(int id,String name)
	{
		this.id=id;
		this.name=name;
	}
	
	public String toString()
	{
		return "Employee ID: "+id+"\nEmployee Name: "+name;
	}
}
